code/game/machinery/quantum_pad.dm

```diff
@@ -15,6 +15,16 @@
 	var/teleporting = 0 //if it's in the process of teleporting
 	var/power_efficiency = 1
 	var/obj/machinery/quantumpad/linked_pad
+	
+	//mapping
+	var/static/list/mapped_quantum_pads = list()
+	var/map_pad_id = "" as text //what's my name
+	var/map_pad_link_id = "" as text //who's my friend
+	
+/obj/machinery/quantumpad/Initialize()
+	. = ..()
+	if(map_pad_id)
+		mapped_quantum_pads[map_pad_id] = src
```

Comment:
This ref needs to be removed in Destroy()

Reply:
sure, It was late at the time.